main.js

```diff
@@ -52,7 +52,30 @@ const menubarApp = menubar({
 });
 
 menubarApp.on('ready', () => {
+  // Utility function to reposition the window so it stays in the tray center
+  function moveToTrayCenter() {
```

Comment:
Why do the function definitions need to be in the ready scope? Couldn't they be at the top level scope? Would make for cleaner code that's easier to refactor.